_includes/downloadsInclude.txt

```diff
@@ -4,8 +4,9 @@
 |:---------:|:-------:|:---:|:---:|:------:|:---------:|:------:|:---------:|
 | Java Core | 6.1.0 | [Download](https://www.apache.org/dyn/closer.lua/datasketches/java/6.1.0/apache-datasketches-java-6.1.0-src.zip) | [Signature](https://downloads.apache.org/datasketches/java/6.1.0/apache-datasketches-java-6.1.0-src.zip.asc) | [Hash](https://downloads.apache.org/datasketches/java/6.1.0/apache-datasketches-java-6.1.0-src.zip.sha512) | 2024-09-14T18:02:32Z | 71586 | leerho |
 | C++ Core | 5.1.0 | [Download](https://www.apache.org/dyn/closer.lua/datasketches/cpp/5.1.0/apache-datasketches-cpp-5.1.0-src.zip) | [Signature](https://downloads.apache.org/datasketches/cpp/5.1.0/apache-datasketches-cpp-5.1.0-src.zip.asc) | [Hash](https://downloads.apache.org/datasketches/cpp/5.1.0/apache-datasketches-cpp-5.1.0-src.zip.sha512) | 2024-08-02T23:02:28Z | 70672 | alsay |
+| Python Core | 5.0.2 | [Download](https://www.apache.org/dyn/closer.lua/datasketches/python/5.0.2/apache-datasketches-python-5.0.2-src.zip) | [Signature](https://downloads.apache.org/datasketches/python/5.0.2/apache-datasketches-python-5.0.2-src.zip.asc) | [Hash](https://downloads.apache.org/datasketches/python/5.0.2/apache-datasketches-python-5.0.2-src.zip.sha512) | 2024-04-08T20:20:47Z | 68380 | jmalkin |
```

Comment:
Please manually remove this line. Realized I forgot to remove from svn, which I will do momentarily.

Reply:
Removed.